examples/docking-protein-DNA/docking-protein-DNA-mdref.cfg

```diff
@@ -22,34 +22,36 @@ molecules =  [
 autohis = true
 
 [rigidbody]
-ambig_fname = 'data/cro_air.tbl'
+ambig_fname = "data/cro_air.tbl"
```

Comment:
FYI, haddock3 config reader accepts both single and double quotes

Reply:
For consistency I switched everywhere to double quotes